utils/mail.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require("nodemailer");
+
+const transporter = nodemailer.createTransport({
+  host: "smtp.mailgun.org",
+  port: 587,
+  auth: {
+    user: "postmaster@mailgun.org",
+    pass: process.env.MAIL_API_KEY,
+  },
+});
+
+const sendVerification = async (email, verificationToken) => {
+  const msg = {
+    from: "postmaster@sandbox15195c20faeb470ca655478a9f22042c.mailgun.org",
+    to: email,
+    subject: `Verification for Registration`,
+    html: `
+    <p>Click below to verify:
+    <a href="http://localhost:3000/api/users/verify/${verificationToken}">Verify Email</a></p>
```

Comment:
Zawsze można zastanowić się czy jakiegoś ładniejszego maila do tej osoby nie wysłać :) Ale to na przyszłość sugestia.